.env.example

```diff
@@ -0,0 +1,22 @@
+# Logging
+DEBUG=info*,error*,warn*,debug*
+# keys are derived from the mnemonic above
+DEPLOYER_PRIVATE_KEY=0x754fde3f5e60ef2c7649061e06957c29017fe21032a8017132c0078e37f6193a
+SEQUENCER_PRIVATE_KEY=0xd2ab07f7c10ac88d5f86f1b4c1035d5195e81f27dbe62ad65e59cbf88205629b
+TX_INGESTION_SIGNER_KEY=d2ab07f7c10ac88d5f86f1b4c1035d5195e81f27dbe62ad65e59cbf88205629b
```

Comment:
There are a few extra env vars being included here, we should make sure to only include the ones that are being used by this service. You can see the required + optional env vars here: https://github.com/ethereum-optimism/batch-submitter/blob/a83d218ede779e42c5f3dbee93aa976f9a1275a7/src/exec/run-batch-submitter.ts#L58